README.md

```diff
@@ -308,3 +308,21 @@ The following is equivalent:
 
   See more on [this blog post](https://medium.com/@tomchentw/imagemin-lint-staged-in-place-minify-the-images-before-adding-to-the-git-repo-5acda0b4c57e) for benefits of this approach.
 </details>
+
+### :warning: Important note for users of JetBrains IDEs *(WebStorm, PyCharm, IntelliJ IDEA, RubyMine, etc.)*
```

Comment:
Let's introduce a H2 (`##`) section named `Frequently Asked Questions` so it's more structured.

The title could also be simplified to something like `Using with JetBrains IDEs` and remove the warning emoji (nice one, BTW)